### ISSUE TITLE
Fasta assembly not recognized

### ISSUE THREAD

**Issue by alexvasilikop** [RolandFaure/Hairsplitter#1]
Hello,

Thanks for developing such useful tools.

For some reason my assembly file is not recognized:
```
python3 hairsplitter.py -i $assembly -f $reads -x ont -t 12 -o $outdir

	******************
	*                *
	*  Hairsplitter  *
	*    Welcome!    *
	*                *
	******************

ERROR: Assembly file must be in GFA or FASTA format. File extension not recognized.
```

I have tried with both ```.fasta``` and ```.fa``` extensions. Any idea what the issue is?
I am using 
```
HairSplitter v1.3.0 (RolandFaure/HairSplitter). Last update: 2023-07-20
```

Thanks
Alex

**Comment by alexvasilikop** [comment]
Ok apparently the issue was that it most likely expects a ```.gfa``` file. When I renamed the fasta file to have a ```.gfa``` extension then it worked. Perhaps something to improve in the future.

cheers

**Comment by RolandFaure** [comment]
Hi Alex,
This was a bug occuring when handling the fasta files. Thank you very much for bringing it up, the bug has been corrected in the new version 1.3.1. 
If you ever need to convert fasta to gfa, do not simply change the name of the file, this will confuse HairSplitter. Actually transform the fasta into GFA (with 'S' lines only).

Do not hesitate if you have other issues :smiley:
